packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -70,10 +70,7 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
   const hashFormat = getOutputHashFormat(buildOptions.outputHashing || 'none');
   const buildBrowserFeatures = new BuildBrowserFeatures(projectRoot);
 
-  const targetInFileName = getEsVersionForFileName(
-    tsConfig.options.target,
-    buildOptions.differentialLoadingNeeded,
-  );
+  const targetInFileName = getEsVersionForFileName(tsConfig.options.target, false);
```

Comment:
This line and the helper function can be removed since the second argument is false.

Reply:
Removed.